bucket/srecord.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please reorder tags per this template: https://github.com/Ash258/Scoop-Ash258/blob/master/.vscode/Template.yml